packages/core/src/browser/dialogs.ts

```diff
@@ -149,7 +149,7 @@ export abstract class AbstractDialog<T> extends BaseWidget {
     }
 
     protected addCloseAction<K extends keyof HTMLElementEventMap>(element: HTMLElement, ...additionalEventTypes: K[]): void {
-        this.addKeyListener(element, Key.ESCAPE, () => this.close(), ...additionalEventTypes);
+        this.addKeyListener(element, Key.ENTER, () => this.close(), ...additionalEventTypes);
```

Comment:
why did you change this to enter?

Reply:
it was a bug, close button should be triggered on `enter` when it is focused, not on `esc`

Reply:
ok